example-playbooks/replace_node/replace_node.yml

```diff
@@ -195,3 +195,39 @@
             _rbno_allowed_for_replace: |
               {% if (_scylla_yaml_map.allowed_repair_based_node_ops is not defined or 'replace' in _scylla_yaml_map.allowed_repair_based_node_ops) %}True{% else %}False{% endif %}
       when: _enable_rbno_grep.failed == false
+
+
+# This play will repair the new node if RBNO was not used and skip_repair is not set to true
+- name: Repair the new node, if necessary
+  hosts: scylla-manager
+  become: false
+  tasks:
+    - name: Skip this playbook if RBNO was used for the replacement or if skip_repair is true
+      meta: end_host
+      when: skip_repair|bool or (hostvars[new_node]['_enable_rbno_grep'].failed == false and hostvars[new_node]['_rbno_enabled'] and hostvars[new_node]['_rbno_allowed_for_replace'])
```

Comment:
Don't you need to use `|bool` for every boolean you check to be `True` in this expression?

Reply:
No, we only need to cast if we're not sure that the variable is boolean.
For example, for the variable `skip_repair` the user could pass `skip_repair: 'false'` instead of `skip_repair: false`, so it would make sense to cast the variable to boolean.
But in this specific case we know that `failed`, `_rbno_enabled` and `_rbno_allowed_for_replace` are always a boolean.